src/utils/azureUtils.ts

```diff
@@ -115,115 +100,43 @@ export async function getArmTagKeys(context: IActionContext): Promise<Set<string
-    'microsoft.network/networksecuritygroups': { displayName: localize('networkSecurityGroups', 'Network security groups') },
-    'microsoft.network/loadbalancers': { displayName: localize('loadBalancers', 'Load balancers') },
-    'microsoft.compute/disks': { displayName: localize('disks', 'Disks') },
-    'microsoft.compute/images': { displayName: localize('images', 'Images') },
-    'microsoft.compute/availabilitysets': { displayName: localize('availabilitySets', 'Availability sets') },
-    'microsoft.compute/virtualmachinescalesets': { displayName: localize('virtualMachineScaleSets', 'Virtual machine scale sets') },
-    'microsoft.network/virtualnetworks': { displayName: localize('virtualNetworks', 'Virtual networks') },
-    'microsoft.cdn/profiles': { displayName: localize('frontDoorAndcdnProfiles', 'Front Door and CDN profiles') },
-    'microsoft.network/publicipaddresses': { displayName: localize('publicIpAddresses', 'Public IP addresses') },
-    'microsoft.network/networkinterfaces': { displayName: localize('networkInterfaces', 'Network interfaces') },
-    'microsoft.network/networkwatchers': { displayName: localize('networkWatchers', 'Network watchers') },
-    'microsoft.batch/batchaccounts': { displayName: localize('batchAccounts', 'Batch accounts') },
-    'microsoft.containerregistry/registries': { displayName: localize('containerRegistry', 'Container registry') },
-    'microsoft.dbforpostgresql/servers': { displayName: localize('postgreSqlServers', 'PostgreSQL servers (Standard)') },
-    'microsoft.dbforpostgresql/flexibleservers': { displayName: localize('postgreSqlServers', 'PostgreSQL servers (Flexible)') },
-    'microsoft.dbformysql/servers': { displayName: localize('mysqlServers', 'MySql servers') },
-    'microsoft.sql/servers/databases': { displayName: localize('sqlDatabases', 'SQL databases') },
-    'microsoft.sql/servers': { displayName: localize('sqlServers', 'SQL servers') },
-    'microsoft.documentdb/databaseaccounts': { displayName: localize('documentDB', 'Azure Cosmos DB') },
-    'microsoft.operationalinsights/workspaces': { displayName: localize('operationalInsightsWorkspaces', 'Operational Insights workspaces') },
-    'microsoft.operationsmanagement/solutions': { displayName: localize('operationsManagementSolutions', 'Operations management solutions') },
-    'microsoft.insights/components': { displayName: localize('insightsComponents', 'Application Insights') },
-    'microsoft.web/serverfarms': { displayName: localize('serverFarms', 'App Service plans') },
-    'microsoft.web/kubeenvironments': { displayName: localize('containerService', 'App Service Kubernetes Environment') },
-    'microsoft.app/managedenvironments': { displayName: localize('containerAppsEnv', 'Container Apps Environment') },
-    'microsoft.app/containerapps': { displayName: localize('containerApp', 'Container Apps') },
+const azExtDisplayInfo: Partial<Record<AzExtResourceType, AzExtResourceTypeDisplayInfo>> = {
```

Comment:
Is `Partial` needed? Or is that because not all `AzExtResourceType`s are included?

Reply:
> because not all AzExtResourceTypes are included?

Correct